### PR TITLE
Items split

### PR DESCRIPTION
**Pull request by mehrdadgit**
into OC-22 branch

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -20,22 +20,34 @@ CREATE TYPE tba21.licence_type AS ENUM ('CC BY', 'CC BY-SA', 'CC BY-ND', 'CC BY-
...
+-- S3 objects metadata table
+CREATE TABLE tba21.s3uploads
+(
+	sha512 varchar(128) PRIMARY KEY,
+	all_s3_keys varchar[] , --all duplicate keys, if any, go here
+	created_at timestamp with time zone NOT NULL,
+	updated_at timestamp with time zone NOT NULL,
+    exif jsonb, -- for things that don't go into other columns
+	machine_recognition_tags varchar[],
+	other_metadata jsonb -- anything else like image-hash etc ? could decompose them here too
```
Comment (matthewberryman) [inline schema.sql]: I think image hash should go separately. AFAIK only other metadata is exif (which we already have at L30) so we could drop other_metadata and just have image_hash

File: schema.sql [hunk 2]
```diff
@@ -20,22 +20,34 @@ CREATE TYPE tba21.licence_type AS ENUM ('CC BY', 'CC BY-SA', 'CC BY-ND', 'CC BY-
...
+	s3_key varchar , --s3 key, if any, associated to this particular item
 	created_at timestamp with time zone NOT NULL,
 	updated_at timestamp with time zone NOT NULL,
 	time_produced timestamp with time zone,
 	status boolean,
-	exif jsonb, -- for things that don't go into other columns
 	concept_tags bigint[],
 	keyword_tags bigint[],
-	recognition_tags varchar(128),
+	recognition_tags varchar[],
```
Comment (matthewberryman) [inline schema.sql]: yes good catch

File: schema.sql [hunk 3]
```diff
@@ -102,8 +118,8 @@ CREATE INDEX collections_gix ON tba21.collections USING GIST (geom); -- collecti
 -- Collection items cross-references
 CREATE TABLE tba21.collections_items
 (
-	ID bigint references collections(id),
-	sha512 varchar(128) references items(id)
+	collection_ID bigint references collections(ID),
```
Comment (matthewberryman) [inline schema.sql]: clearer - thanks

File: schema.sql [hunk 4]
```diff
@@ -29,7 +29,7 @@ CREATE TABLE tba21.s3uploads
 	updated_at timestamp with time zone NOT NULL,
     exif jsonb, -- for things that don't go into other columns
 	machine_recognition_tags varchar[],
-	other_metadata jsonb -- anything else like image-hash etc ? could decompose them here too 
+	md5_hash varchar(32)
```
Comment (matthewberryman) [inline schema.sql]: also need image hash

File: schema.sql (hunk 1 above)
Comment (mehrdadgit) [reply schema.sql]: Fixed; added md5 just in case we want to keep it from helpers

File: schema.sql (hunk 4 above)
Comment (mehrdadgit) [reply schema.sql]: Oh yes; image hash in now

Comment (matthewberryman) [reply schema.sql]: Cool just saw your other commit pushed. Thx

**Pull request merged**